centaur/src/main/scala/centaur/test/Test.scala

```diff
@@ -220,42 +220,7 @@ object Operations {
   }
 
   def validatePAPIAborted(jobId: String, workflow: SubmittedWorkflow): Test[Unit] = {
-    import scala.concurrent.duration._
-
-    def eventually(startTime: OffsetDateTime, timeout: FiniteDuration)(f: => Try[Unit]): Try[Unit] = {
-      f match {
-        case Failure(_) if OffsetDateTime.now().isBefore(startTime.plusSeconds(timeout.toSeconds)) =>
-          blocking { Thread.sleep(5.seconds.toMillis) }
-          eventually(startTime, timeout)(f)
-        case t => t
-      }
-    }
-    
     new Test[Unit] {
-      def checkVMTerminated(): Unit = {
```

Comment:
Turns out this was never really testing the right thing. The jes instance name is not always going to be there if the VM didn't have time to start. It's already checking that JES has aborted the job which is all we can do anyway.